apps/events/views.py

```diff
@@ -402,20 +422,12 @@ def post(self, request, format=None):
         username = request.data.get('username')
         waitlist_approved = request.data.get('approved')
 
-        if not (username or rfid):
-            return Response({'message': 'Mangler både RFID og brukernavn. Vennligst prøv igjen.',
-                             'attend_status': 41}, status=status.HTTP_400_BAD_REQUEST)
-
-        # If attendee has typed in username to bind a new card to their user
-        if username is not None and rfid is not None:
-            try:
-                user = User.objects.get(username=username)
-                user.rfid = rfid
-                user.save()
-            except User.DoesNotExist:
-                return Response({'message': 'Brukernavnet finnes ikke. Husk at det er et online.ntnu.no brukernavn! '
-                                            '(Prøv igjen, eller scan nytt kort for å avbryte.)', 'attend_status': 50},
-                                status=status.HTTP_400_BAD_REQUEST)
+        error = validate_attend_params(rfid, username)
+        if 'message' in error and 'attend_status' in error:
+            return Response({'message': error.get('message'), 'attend_status': error.get('attend_status')},
+                            status=status.HTTP_400_BAD_REQUEST
+                            )
```

Comment:
Formatting 🤢

Reply:
What's wrong?

Reply:
It's probably fine by pep8 standards and just me being weird.
It was just the combination of a bunch of spaces to match indention level of the line above and adding the ending parenthesis on a new line with the same indention level. 

In cases like this I prefer 
```python
return Response(
    {'message': error.get('message'), 'attend_status': error.get('attend_status')},
    status=status.HTTP_400_BAD_REQUEST
)
```

or (but this is probably not allowed by pep8 idk)

```python
return Response({'message': error.get('message'), 'attend_status': error.get('attend_status')},
    status=status.HTTP_400_BAD_REQUEST)
```

Either way feel free to ignore the comment.